### ISSUE TITLE
Update Timeliness gem to close file vulnerability gap 

### ISSUE THREAD

**Issue by ChrisMcGowan** [cloudfoundry/cloud_controller_ng#1725]
## Issue

Using an old (2016) Ruby Gem for Timeliness that writes files out to the file system as world writable `0666`.  This was fixed in later releases of the gem.

## Context

The timeliness 0.3.8 gem used in capi: (https://github.com/cloudfoundry/cloud_controller_ng/blob/7b9625595651fa59d5a93454b3fbff3bb5440848/Gemfile.lock#L459) writes out files to the filesystem on a api instance VM with world writable permissions `0666`.  The gem author has fixed this in version of 0.3.10 and higher of the gem: https://github.com/adzap/timeliness/blob/master/CHANGELOG.rdoc

## Steps to Reproduce

SSH into any of the api VMs and as sudo go to `/var/vcap/data/packages/cloud_controller_ng/<bosh random package guid>/gem_home/ruby/2.5.0/gems/timeliness-0.3.8` and look at the file permissions

## Expected result

That the gem is updated to a more current version where world permissions are removed from the gem files.

## Current result

Sample `ls -al` listing:

```
drwxr-xr-x   4 root root  4096 Jul  1 18:33 ./
drwxr-xr-x 144 root root  4096 Jul  1 18:34 ../
-rw-rw-rw-   1 root root  3707 Jul  1 18:33 benchmark.rb
-rw-rw-rw-   1 root root  1598 Jul  1 18:33 CHANGELOG.rdoc
drwxr-xr-x   3 root root  4096 Jul  1 18:33 lib/
-rw-rw-rw-   1 root root  1055 Jul  1 18:33 LICENSE
-rw-rw-rw-   1 root root   610 Jul  1 18:33 Rakefile
-rw-rw-rw-   1 root root 10475 Jul  1 18:33 README.rdoc
drwxr-xr-x   3 root root  4096 Jul  1 18:33 spec/
-rw-rw-rw-   1 root root  1053 Jul  1 18:33 timeliness.gemspec
-rw-rw-rw-   1 root root   104 Jul  1 18:33 .travis.yml
```

## Possible Fix

Upgrade to use gem `0.3.10` or higher

**Comment by pburkholder** [comment]
Although I can't find the commit that fixed the file permissions issues at https://github.com/adzap/timeliness/compare/v0.3.8...v0.3.10, I did install both 0.3.8 and 0.3.10 locally and confirmed that 0.3.10 has fixed the file permissions issue.

**Comment by selzoc** [comment]
Bumped in ab03382acd0d52f7bcb7ca70e97dff0c6085b938

**Comment by Gerg** [comment]
This is now available in [capi-release 1.97](https://github.com/cloudfoundry/capi-release/releases/tag/1.97.0)
